uint32 date
uint32 time
float64 longitude_RTK
float64 latitude_RTK
float32 height_above_sea_RTK
float32 velocity_north
float32 velocity_east
float32 velocity_ground
int16 yaw #between baseline and south
uint8 position_flag
uint8 yaw_flag
